test/index-test.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
Hmm, `.eslintrc` should be a little bit updated, I reckon.

![20150617-102541](https://cloud.githubusercontent.com/assets/175264/8197415/44946b44-14db-11e5-8e20-9c6ec63b62c3.png)